Gulpfile.js

```diff
@@ -74,78 +67,104 @@ gulp.task('copy', ['sass'], function () {
       .pipe(gulp.dest('build/assets')),
     // minify requirejs
     gulp.src(['build/vendor/requirejs/require.js'])
-      .pipe(uglify().on('error', handleError))
+      .pipe(plumber(handleError))
+      .pipe(uglify())
       .pipe(gulp.dest('build/vendor/requirejs'))
```

Comment:
For some reason I don't have requirejs minified in a build now (can be a race condition). I came up with this fix locally:

```
    gulp.src(['source/vendor/**/*', '!source/vendor/requirejs/require.js'])
      .pipe(gulp.dest('build/vendor')),
    // minify requirejs
    gulp.src(['source/vendor/requirejs/require.js'])
      .pipe(plumber(handleError))
      .pipe(uglify())
      .pipe(gulp.dest('build/vendor/requirejs'))
  );
```